### ISSUE TITLE
Properly handle columns with mixed numeric and categorical values

### ISSUE THREAD

**Issue by navado** [pandas-profiling/pandas-profiling#70]
You might want to handle them as categorical or define parameter to solve the following problem:
```
  File "/home/user/.local/lib/python3.5/site-packages/pandas_profiling/__init__.py", line 17, in __init__
    description_set = describe(df, **kwargs)
  File "/home/user/.local/lib/python3.5/site-packages/pandas_profiling/base.py", line 308, in describe
    confusion_matrix=pd.crosstab(data1,data2)
  File "/home/user/.local/lib/python3.5/site-packages/pandas-0.20.3-py3.5-linux-x86_64.egg/pandas/core/reshape/pivot.py", line 493, in crosstab
    aggfunc=len, margins=margins, dropna=dropna)
  File "/home/user/.local/lib/python3.5/site-packages/pandas-0.20.3-py3.5-linux-x86_64.egg/pandas/core/reshape/pivot.py", line 160, in pivot_table
    table = table.sort_index(axis=1)
  File "/home/user/.local/lib/python3.5/site-packages/pandas-0.20.3-py3.5-linux-x86_64.egg/pandas/core/frame.py", line 3243, in sort_index
    labels = labels._sort_levels_monotonic()
  File "/home/user/.local/lib/python3.5/site-packages/pandas-0.20.3-py3.5-linux-x86_64.egg/pandas/core/indexes/multi.py", line 1241, in _sort_levels_monotonic
    indexer = lev.argsort()
  File "/home/user/.local/lib/python3.5/site-packages/pandas-0.20.3-py3.5-linux-x86_64.egg/pandas/core/indexes/base.py", line 2091, in argsort
    return result.argsort(*args, **kwargs)
TypeError: unorderable types: str() < int()
```

**Comment by romainx** [comment]
Please could you provide an example permitting to reproduce the case?

Note: Since #43, it is possible to call the report with the option `check_correlation=False` to skip this phase. In your case, it will avoid the error since it appears during the correlation computing.

**Comment by navado** [comment]
Example data: [ex.txt](https://github.com/JosPolfliet/pandas-profiling/files/1469977/ex.txt)
More observations - if I reduce the size of the example to ~1000 points, including the problematic records, the problem does not happen. 
Another strange thing, enabling `check_correlation=False` throws an error
```
>>> pfr = pandas_profiling.ProfileReport(subset,check_correlation=False)
.....
TypeError: _plot_histogram() got an unexpected keyword argument 'check_correlation'
```

**Comment by conradoqg** [comment]
Hey,

This issue was fixed in the PR #82.

I think we can close this issue after those changes.

Best
